cardano-crypto-tests/src/Test/Crypto/DSIGN.hs

```diff
@@ -126,8 +134,10 @@ tests =
     [ testDSIGNAlgorithm mockSigGen (arbitrary @Message) "MockDSIGN"
     , testDSIGNAlgorithm ed25519SigGen (arbitrary @Message) "Ed25519DSIGN"
     , testDSIGNAlgorithm ed448SigGen (arbitrary @Message) "Ed448DSIGN"
+#ifdef SECP256K1
     , testDSIGNAlgorithm secp256k1SigGen genSECPMsg "EcdsaSecp256k1DSIGN"
     , testDSIGNAlgorithm schnorrSigGen (arbitrary @Message) "SchnorrSecp256k1DSIGN"
+#endif
```

Comment:
@michaelpj Honestly, there is a little too much CPP for my taste, I'd put all secp256k1 related tests into a separate module and imported it with a single CPP `#if` statement. 

However, I am not gonna stand in a way of correct code, so it's up to you.

Reply:
I'm happy either way, I just wanted to go with the dumbest thing I could get working for now...

We'd still need at least two `#if`s: one for the import and one for the test invocation, but it would indeed be less.

If you don't mind, we can merge this to unblock people and I can make a follow-up with less CPP.